docs/markdown-v2.md

```
@@ -0,0 +1,45 @@
+// Using the Fluent API
+const message2 = new FormattedString("Welcome!")
+  .bold()
+  .append("\nCheck out our ")
+  .append(new FormattedString("website").link("https://grammy.dev"));
+```
+
+## Escaping
+
+When using `fmt` or `FormattedString`, the library handles all necessary escaping for MarkdownV2 automatically. You do not need to manually escape special characters like `_`, `*`, `[`, `]`, `(`, `)`, `~`, `>`, `#`, `+`, `-`, `=`, `|`, `{`, `}`, `.`, `!`. 
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)